dev/Repeater/SelectionModel.cpp

```diff
@@ -619,32 +635,56 @@ void SelectionModel::SelectWithGroupImpl(int groupIndex, int itemIndex, bool sel
 void SelectionModel::SelectWithPathImpl(const winrt::IndexPath& index, bool select, bool raiseSelectionChanged)
 {
     bool selected = false;
+    bool selectionChanged = true;
     if (m_singleSelect)
```

Comment:
>m_singleSelect [](start = 8, length = 14)

Why is the single select case handled specially here, wouldn't the multi select path handle this just fine? Is it just a perf optimization?

Reply:
Yes, this is faster than walking over the actual selection tree. The multi select behavior would also handle this case, but comparing the IndexPaths is faster here.